ide/bugtracking/test/unit/src/org/netbeans/modules/bugtracking/team/DummyRepositories.java

```diff
@@ -17,9 +17,8 @@
  * under the License.
  */
 
-package org.netbeans.modules.bugtracking.kenai;
+package org.netbeans.modules.bugtracking.team;
```

Comment:
```suggestion
package org.netbeans.modules.bugtracking.dummies;
```

This is a hunch - but in `RepositoryComboSupportTest` all other dummy implementations reside in the `dummies` package.